test/http_test.js

```diff
@@ -101,7 +101,8 @@ describe("request()", () => {
         status: 200,
         headers: {
           get(name) {
-            return "fake";
+            if (name !== "Alert")
+              return "fake";
```

Comment:
You could also always return an empty string (since the scope of this test is about response anyway)